Makefile

```diff
@@ -145,6 +145,10 @@ compile: ## Compile ebpf agent project
 	@echo "### Compiling project"
 	GOARCH=${GOARCH} GOOS=$(GOOS) go build -mod vendor -a -o bin/netobserv-ebpf-agent cmd/netobserv-ebpf-agent.go
 
+.PHONY: build-and-push-bc-image
```

Comment:
We should also update the automation / CI for that, right?
- github action
- konflux/tekton

Reply:
Also, our other images are multi-arch / manifest, here it seems it only builds for the current detected architecture, right? I guess we should also create a manifest multi-arch for this one

Reply:
this script takes the byte code aritifcate we already generate from `make docker-generate` doesn't really build, will add GH action konflux for bpfman byte code not necesserly for this TP